app/templates/faq.hbs

```diff
@@ -16,40 +16,62 @@
     <h5>Full integration</h5>
-    <p class="integration-type-text">Programmatic submission from PASS to NIH Manuscript Submission System (NIHMS) and status updates</p>
+    <p class="integration-type-text">Programmatic submission from PASS to NIH Manuscript Submission System (NIHMS) and submission status updates</p>
     <ul>
       <li>NIH - National Institutes of Health</li>
     </ul>
     <h5>Deposit integration</h5>
-    <p class="integration-type-text">Programmatic submission from PASS to NIHMS without status updates</p>
+    <p class="integration-type-text">Programmatic submission from PASS to NIHMS without submission status updates</p>
     <ul>
       <li>ACL - Administration for Community Living</li>
       <li>AHRQ - Agency for Healthcare Research and Quality</li>
-      <li>ASPR –  Office of the Assistant Secretary for Preparedness and Response</li>
+      <li>ASPR – Office of the Assistant Secretary for Preparedness and Response</li>
       <li>CDC – Centers for Disease Control</li>
       <li>VA – Department of Veteran Affairs</li>
       <li>FDA - Food and Drug Administration</li>
       <li>HHMI- Howard Hughes Medical Institute</li>
       <li>NASA -National Aeronautics and Space Administration </li>
     </ul>
     <h5>Linked web portal</h5>
-    <p class="integration-type-text">Programmatic integration is not currently possible, but PASS does provide link to the agencies’ submission portal and provide full list of relevant metadata to aid with submission process</p>
+    <p class="integration-type-text">While programmatic integration is not currently possible, PASS does provide a link to these agencies’ submission portal and provides a list of relevant metadata to aid with submission process</p>
     <ul>
       <li>ED – Department of Education</li>
       <li>USAID – US Agency for International Development</li>
     </ul>
   </section>
   <section>
-    <h3 id="institution-policy">What is my institution's open access policy?</h3>
-    <p>As of July 1, 2018, Johns Hopkins University have put in effect a university wide open access policy that requires all scholarly articles produced by full-time faculty members be made openly available. To learn more about this policy visit the <a href="https://provost.jhu.edu/about/open-access/" target="_blank">Office of the Provost's webpages on Open Access at Johns Hopkins</a>.</p>
+    <h3 id="institution-policy">What is JHU's open access policy?</h3>
+    <p>As of July 1, 2018, Johns Hopkins University has implemented a university wide open access policy that expects  every scholarly article produced by full-time faculty members of Johns Hopkins University be accessible in an open access repository. To learn more about this policy visit the <a href="https://provost.jhu.edu/about/open-access/" target="_blank">Office of the Provost's webpages on Open Access at Johns Hopkins</a>.</p>
```

Comment:
Double space between 'expects' and 'every'. This renders correctly in a browser, so this does not need to be fixed.

> policy that expects  every scholarly